agent/api/task/task_linux.go

```diff
@@ -56,6 +56,10 @@ const (
 
 	minimumCPUPercent = 0
 	bytesPerMegabyte  = 1024 * 1024
+
+	// The minimum docker version that supports "awslogs-endpoint" log option.
+	// Will update this when we know the version.
+	awslogsEndpointMinimumDockerVersion = "19.03.1-ce"
```

Comment:
It's set both in `task_linux.go` and `task_windows.go`, because Linux and Windows use different docker versions.

I will update this version when the the endpoint option is released in Docker.